.circleci/config.yml

```diff
@@ -79,7 +79,7 @@ jobs:
       - setup_remote_docker
       - run: sudo apt-get update
       - run: sudo apt-get install rpm
-      - run: curl -sL https://git.io/goreleaser | bash -s -- --rm-dist --skip-publish --skip-validate --release-notes <(make echo-release-notes)
+      - run: curl -sL https://git.io/goreleaser | VERSION=v0.118.1 bash -s -- --rm-dist --skip-publish --skip-validate --release-notes <(make echo-release-notes)
```

Comment:
latest version of goreleaser was breaking the build, so I locked the version to the last one that worked.